commander/src/bootstrapping/commands/sdk/link.ts

```diff
@@ -37,7 +37,7 @@ export class LinkCommand extends Command {
 			throw new Error(`Path '${targetSDKFolder as string}' does not exist or access denied.`);
 		}
 
-		const sdkLocalPath = join(__dirname, '../../../../', 'node_modules', 'lisk-sdk');
+		const sdkLocalPath = join(process.cwd(), 'node_modules', 'lisk-sdk');
```

Comment:
this only works if it is run under `myApp`, while it was working regardless of the position